app/src/main/java/org/liberty/android/fantastischmemo/entity/Setting.java

```diff
@@ -330,6 +386,14 @@ public void setQuestionFont(String questionFont) {
         this.questionFont = questionFont;
     }
 
+    public String getHintFont() {
+        return answerFont;
```

Comment:
Are we using the same database fiend for the hint and answer font?

Reply:
the same font yes